site/index.html

```diff
@@ -99,14 +99,14 @@ <h3>Scale with ease</h3>
          <div class=" span6 col-md-6">
             <h3>Maintain with confidence</h3>
             <p>Add complex features or refactor large parts of your code base reliably. Haskell has one of the strongest type systems on the market to prevent mistakes
-               before they reach production, saving on total software lifetime costs.&mdash;“If it compiles, it works.” and "Fearless refactoring" are common proverbs among Haskellers.
+               before they reach production, saving on total software lifetime costs.&mdash;“If it compiles, it works” and "Fearless refactoring" are common proverbs among Haskellers.
             </p>
          </div>
          <div class=" span6 col-md-6">
-            <h3>Cutting edge</h3>
+            <h3>Bleeding edge</h3>
             <p>Get a view into the latest programming innovations. Haskell has its roots in programming language research, has already influenced many popular languages and continues to do so.
-               Try out new programming techniques like Generalized Algebraic Data Types (GADT), Effect Systems, Linear Types and others for even more reliability or performance
-               &mdash; when you're curious or when you want to get ahead.
+               Experiment with new programming techniques like Effect Systems, Linear Types and more related to Dependent Types for even better reliability when you can tolerate some immaturity;
```

Comment:
You could also add Functional Reactive Programming to this list.

Reply:
Done